tutorials/commands/ssh-across-clusters.rst

```diff
@@ -0,0 +1,156 @@
+.. _ssh-across-clusters:
+
+===================
+SSH across clusters
+===================
+
+Let's say you want to create a Flux instance in an allocation on a cluster (e.g., let's say out first cluster is "noodle") 🍜️
+and then connect to it via ssh from another cluster (let's say our second cluster is called quartz). This is possible with the right
+setup of your ``~/.ssh/config``. 
+
+----------------------
+Create a Flux Instance
+----------------------
+
+First, let's create the allocation on the first cluster. We typically want to ask for an allocation,
+and run flux start via our job manager.
+
+.. code-block:: sh
+
+    # slurm specific
+    $ salloc -N4 --exclusive
+    $ srun -N4 -n4 --pty --mpibind=off flux start
```

Comment:
I thought we were on some general noodle node (that is part of the allocation?)

Reply:
okay I added a comment at the top that they are on a login node, and then when they hit "noodle" they have their allocation.

Reply:
My reasoning for the prompt change is b/c of your ssh config below.  `noodle` appears to be the login node, while `noodle220` is the node that you were allocated to run your job.  So it may not be clear which node you're actually on with just `noodle:~$`??

For example, w/ my personal prompt:

```
opal186|/g/g0/achu 44>salloc -N4 -ppbatch
salloc: Granted job allocation 321078
salloc: Waiting for resource configuration
salloc: Nodes opal[63-66] are ready for job

opal63|/g/g0/achu 21>srun hostname
opal63
opal66
opal65
opal64
```

you'll notice I was on opal186, the login node, and then `salloc` dropped me into a shell on `opal63`.  So if I were to setup the ssh config, I would think I should set it up for `opal186` and `opal63`.

Reply:
I changed it so the login node is just empty (no name) and I explicitly state we are on the login node. Then I state we are on the allocation and just use `noodle:~$` to say we are on the allocation (and the specific node largely doesn't matter). I thought it looked nicer without the number so I left it out.